lib/npmdc/errors.rb

```diff
@@ -39,9 +39,20 @@ def banner
         path = options.fetch(:path)
         [
           "Can't find `node_modules` folder inside '#{path}' directory!",
-          ["\nRun `npm install` to install missing packages.", :warn]
+          [command_to_resolve_errors, :warn]
         ]
       end
+
+      private
+
+      def command_to_resolve_errors
```

Comment:
I don't like the idea of putting checker-specific logic here. It'd be better to move this out to the checkers.